### PR TITLE
feat: Svelte added to interests dropdown list

### LINKED ISSUE #2167: Feature: Add Svelte in the list of available languages to filter on
### Type of feature

🍕 Feature

### Current behavior

Svelte is not a filter provided.

### Suggested solution

Add svelte to the dropdown. I will attempt to do this myself.

### Additional context

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [X] I agree to follow this project's Contribution Docs

### PR DESCRIPTION
**Pull request by brittneypostma**
## Description

added svelte to interests dropdown list

## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Related Tickets & Documents

Fixes #2167

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [ ] 🙅 no documentation needed

### CODE REVIEW

**Review by nickytonline — CHANGES_REQUESTED** [review]
Thanks for the pull request @brittneypostma!

It looks like we need to add some info in regards to Svelte elsewhere as well. We need an icon
for it in SVG format. See this file, https://github.com/open-sauced/app/blob/beta/lib/utils/getTopicThumbnail.ts, to add it and for examples for other languages.

**Comment by brittneypostma** [comment]
Thanks Nick, just added the logo, not sure what else you need for it. Just let me know :)

**Comment by nickytonline** [comment]
Looks like it works but there are some type errors failing the build.

4:49:27 PM: ./lib/utils/recommendations.ts:3:7
4:49:27 PM: Type error: Property "svelte" is missing in type "{ react: string[]; javascript:
stringIl; python: string|]; ml: string|]; ai: stringI]; rust: string[l; ruby: string[]; c:
stringIl; cpp: string|]; csharp: string|]; php: string|]; java: string[]; typescript: string|];
golang: string||; vue: string||; kubernetes: string|]; hacktoberfest: string|]; clojure:
stringIl; }" but required in type "Record<"ruby" | "javascript" | "python" | "java" ||
"typescript" | "csharp" | "cpp" | "php" | "c" | "ai" | "ml" | "react" | "golang" | "rust" |
"svelte" | "vue" | "kubernetes" | "hacktoberfest" | "clojure", string[]>".
4:49:27 PM: 1 | import { interestsType } from "./getInterestOptions";
4:49:27 PM: 2
4:49:27 PM: > 3 | const recommendations: Record‹interestsType, string[]> = {
4:49:27 PM: ^
4:49:27 PM: 4 | react: ["Skyscanner/backpack"],
4:49:27 PM: 5 | javascript: ["EddieHubCommunity/LinkFree"],
4:49:27 PM: python: ["randovania/randovania"],
4:49:28 PM: Failed during stage "building site": Build script returned non-zero exit code: 2

**Comment by nickytonline** [comment]
Looks like there’s another compile error. 

```
6:14:54 AM: Failed during stage "building site": Build script returned non-zero exit code: 2
6:14:54 AM: Failed to compile.
6:14:54 AM: 
6:14:54 AM: ./lib/utils/recommendations.ts:3:7
6:14:54 AM: Type error: Property "svelte" is missing in type "{ react: string[]; javascript: string[]; python: string[]; ml: string[]; ai: string[]; rust: string[]; ruby: string[]; c: string[]; cpp: string[]; csharp: string[]; php: string[]; java: string[]; typescript: string[]; golang: string[]; vue: string[]; kubernetes: string[]; hacktoberfest: string[]; clojure: string[]; }" but required in type "Record<"ruby" | "javascript" | "python" | "java" | "typescript" | "csharp" | "cpp" | "php" | "c" | "ai" | "ml" | "react" | "golang" | "rust" | "svelte" | "vue" | "kubernetes" | "hacktoberfest" | "clojure", string[]>".
```

Once that error is sorted, run `npm run build` to see if there are any other TypeScript compilation errors.

**Comment by brittneypostma** [comment]
Yeah, trying to do this in GitPod and not getting much feedback. I'm surprised it's passing the commit lint, but I guess if I'm not touching the file, it doesn't see it. I'm not sure what this file is asking for on the value props here. 
image

**Comment by nickytonline** [comment]
I can take a peek. I'll see what's missing. I'll be live streaming code reviews in an hour as that's how I usually start my Mondays, so we can see what's up.

File: lib/utils/recommendations.ts [hunk 1]
```diff
@@ -19,6 +19,7 @@ const recommendations: Record<interestsType, string[]> = {
   kubernetes: ["kubernetes/kubernetes"],
   hacktoberfest: ["open-sauced/app"],
   clojure: ["clojure/clojurescript"],
+  svelte: ["sveltejs/svelte"],
```
Comment (nickytonline) [inline lib/utils/recommendations.ts:22]: I pushes this change @brittneypostma to fix the build error.

**Review by nickytonline — APPROVED** [review]
This is all working now @brittneypostma. Thanks for the PR!

**Review by brandonroberts — APPROVED** [review]
🚀

**Pull request merged**